core/controllers/profile.py

```diff
@@ -41,6 +41,16 @@
 class ProfileHandler(base.BaseHandler):
     """Provides data for the profile page."""
 
+    HANDLER_ARGS_SCHEMAS = {
```

Comment:
As guided in [wiki page](https://github.com/oppia/oppia/wiki/Schemas#how-to-write-validation-schemas-for-handlers) please use both of the constants -- URL_PATH_ARGS_SCHEMAS & HANDLER_ARGS_SCHEMAS to write schemas.

Also username should be written in URL_PATH_ARGS_SCHEMAS instead of HANDLER_ARGS_SCHEMAS.
For complete understanding kindly read wiki page and make a note on where you faced difficulty in understanding, after reading the wiki page & send this note to me or any other contributor so that we can improve the documentation.
Thanks!